ansible/memcached-operator/watches.yaml

```diff
@@ -3,3 +3,9 @@
   group: cache.example.com
   kind: Memcached
   role: /opt/ansible/roles/Memcached
+
```

Comment:
You should only have one watches file entry per GVK

Reply:
👍